host-provision.yml

```diff
@@ -16,12 +16,12 @@
 
 # Assumption: 
 # First time connection is not through public private key pair
-# FIrst time connection is always through customeradmin credentials password authentication
+# FIrst time connection is always through {{ firsttime_connect_user }} credentials password authentication
```

Comment:
Nit: First, not FIrst

Reply:
Done

Reply:
Pushed a branch, but with this iteration, left the following as it is, as noted in an earlier comment:
 `      prompt: Enter firsttime_connect_user password`

We can revisit it later, if required.

Reply:
MOdified the prompt to:
```yml
      prompt: Enter customeradmin password
```

Reply:
> MOdified the prompt to:
> 
> ```yaml
>       prompt: Enter customeradmin password
> ```

Adding background, it seems that `pause` runs too late in the execution step to be usable for SSH passwords, bringing us back to `vars_prompt`.  Change looks good.